ipykernel/comm/comm.py

```diff
@@ -21,15 +21,22 @@ class Comm(LoggingConfigurable):
     shell = Instance('IPython.core.interactiveshell.InteractiveShellABC',
                      allow_none=True)
     kernel = Instance('ipykernel.kernelbase.Kernel')
-    def _kernel_default(self):
+
+    @default('kernel')
+    def _default_kernel(self):
```

Comment:
The new API enabled a lot of simplifications in ipywidgets but for simple things like this, it is true that the net contribution is positive in terms of lines added/removed. ..